src/index.html

```diff
@@ -4,19 +4,99 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
-    <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
+    <title>MOYO</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="img__logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="/apple"
+              class="nav__link is__active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```